src/connection.cpp

```diff
@@ -140,19 +140,19 @@ namespace oxen::quic
         return 0;
     }
 
-    int recv_rx_key(ngtcp2_conn* conn, ngtcp2_encryption_level level, void* user_data)
+    int recv_rx_key(ngtcp2_conn* /*conn*/, ngtcp2_encryption_level /*level*/, void* /*user_data*/)
```

Comment:
function itself currently unused, behavior currently facilitated by the ngtcp2-via-gnutls implementation